tests/ping.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	expect "github.com/google/goexpect"
+	"google.golang.org/grpc/codes"
+	"k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var (
+	promptPriv   = regexp.MustCompile("\\# ")
+	promptUnPriv = regexp.MustCompile("\\$ ")
+	shellSuccess = regexp.MustCompile("\n0")
+	shellFail    = regexp.MustCompile("\n[1-9].*")
+)
+
+// PingFromVMConsole performs a ping through the provided VMI console.
+// Optional arguments for the ping command may be provided, overwirtting the defaul ones.
+// (default ping options: "-c 1, -w 5")
+// Note: The maximum overall command timeout is 10 seconds.
+func PingFromVMConsole(vmi *v1.VirtualMachineInstance, ipAddr string, args ...string) error {
+	pingString := "ping"
```

Comment:
What about using pure golang ping library so we don't need ping to be installed ? 

https://github.com/sparrc/go-ping

Reply:
This is executed from the vm console, I do not understand what you are asking here.

Reply:
Ahh wait I read it wrong is Ping FROM the console, ok then, you can forget about this.